pages/about.tsx

```diff
@@ -38,17 +38,18 @@ import ListItemWithIcon from 'components/list/ListItemWithIcon'
 import { designColor } from 'theme/theme'
 
 import MastheadWithImage from 'components/MastheadWithImage'
-import AboutUsImage from '../assets/aboutUs.png'
-import AppleLogo from '../assets/aboutUsIcons/apple.svg'
-import GeneticsIcon from '../assets/aboutUsIcons/genetics.svg'
-import GoogleLogo from '../assets/aboutUsIcons/google.svg'
-import HCIIcon from '../assets/aboutUsIcons/hci.svg'
-import MicrosoftLogo from '../assets/aboutUsIcons/microsoft.svg'
-import NOAALogo from '../assets/aboutUsIcons/noaa.svg'
-import NonProfitManagementIcon from '../assets/aboutUsIcons/non_profit_management.svg'
-import PhysicsIcon from '../assets/aboutUsIcons/physics.svg'
-import StarbucksLogo from '../assets/aboutUsIcons/starbucks.svg'
-import VerizonLogo from '../assets/aboutUsIcons/verizon.svg'
+
+const AboutUsImage = '/images/aboutUs.png'
```

Comment:
Just curious - why the shift from import to const? I did a little research and couldn't find reason not to other than consistency with other imports.

Reply:
Import from **public** won't resolve.  Since we just need the path, I turned it into a const.